### DISCUSSION TITLE
How to use aggrid enterprise features？

### DISCUSSION THREAD

**Discussion by wasabismile** [zauberzeug/nicegui#3062] [Q&A] [answered]
### Question

I have enterprise lisence key，how can I use the enterprise version features in NiceGUI
![D302A229-2A4C-45bb-BECF-02B4D34792B9](https://github.com/zauberzeug/nicegui/assets/68538468/91334787-5dd8-4ade-87b7-fcc09d8da894)

**Comment by rodja** [comment, marked as answer]
Hello @wasabismile. This has been answered in #3048 and #2431: there is currently no aggrid enterprise support.
